server.go

```diff
@@ -24,113 +26,140 @@ type Route struct {
 	Params map[string]string `yaml:"-"`
 }
 
+// Routes is a list of Routes.
 type Routes struct {
 	Routes []Route `yaml:"routes"`
 }
 
-type ByPathLength []Route
+type byPathLength []Route
 
-func (b ByPathLength) Len() int           { return len(b) }
-func (b ByPathLength) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
-func (b ByPathLength) Less(i, j int) bool { return len(b[i].Path) > len(b[j].Path) }
+func (b byPathLength) Len() int           { return len(b) }
+func (b byPathLength) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
+func (b byPathLength) Less(i, j int) bool { return len(b[i].Path) > len(b[j].Path) }
 
-const TIMEOUT_MILLISECONDS = 500
+const timeout = 500 * time.Millisecond
```

Comment:
In go, a duration value can be constant. If you do not put the type in the definition, it gets more flexible. Now, it is easy to set another unit like 

```go
const timeout = 3 * time.Second
```